Subject: Allow serial console resize to beyond 80 columns.
From: tuksgig tuksgig@gmail.com Sat Mar 23 23:17:14 2024 +0200
Date: Tue Mar 26 17:34:50 2024 +0100:
Git: 4ffbc0d904439aa5d5f818a2ece23bf95969ae98

Replace Gtk.Box.add() with Gtk.Box.pack_start() to place the VTE terminal widget. This enables horizontal expansion/filling the terminal widget when resizing the serial console window.
diff --git a/virtManager/details/serialcon.py b/virtManager/details/serialcon.py
index 8a87c92f..3b479b94 100644
--- a/virtManager/details/serialcon.py
+++ b/virtManager/details/serialcon.py
@@ -273,7 +273,7 @@ class vmmSerialConsole(vmmGObject):
 
         if self._vteterminal:
             scrollbar.set_adjustment(self._vteterminal.get_vadjustment())
-            align.add(self._vteterminal)
+            align.pack_start(self._vteterminal, True, True, 0)
 
         evbox.add(align)
         terminalbox.pack_start(evbox, True, True, 0)
